packages/data-point/lib/entity-types/entity-hash/reducer.js

```diff
@@ -4,67 +4,35 @@ const _ = require('lodash')
 const Promise = require('bluebird')
 const utils = require('../../utils')
 
-function flattenResult (key) {
-  return function flatten (result) {
-    return {
-      key,
-      value: result.value
-    }
-  }
+function resolveMapKeys (accumulator, reducer, resolveTransform) {
+  const transform = { reducers: [reducer] }
```

Comment:
smart :)

Reply:
`{ reducers: [reducer] }` feels a little hacky tbh. I don't like that I'm wrapping the reducer in both an array and an object just so I can pass it to `resolveTransform`